source/material_model/grain_size.cc

```diff
@@ -1042,10 +1042,10 @@ namespace aspect
                              "Units: \\si{\\meter}.");
           prm.declare_entry ("Use paleowattmeter", "true",
                              Patterns::Bool (),
-                             "A flag indicating whether the computation should be use the "
-                             "paleowattmeter approach of Austin and Evans (2007) for grain size reduction "
-                             "in the dislocation creep regime (if true) or the paleopiezometer approach "
-                             "from Hall and Parmetier (2003) (if false).");
+                             "A flag indicating whether the computation should be use the paleowattmeter approach of "
+                             "Austin and Evans (2007) \\url{http://dx.doi.org/10.1130/G23244A.1} for grain size reduction "
```

Comment:
I think a better approach would be to put the full reference into `doc/manual/manual.bib` and `\\cite{...}` it here. Same for Hall and Parmentier a couple of lines below.